app.js

```diff
@@ -0,0 +1,13 @@
+const express=require('express')
+const { urlencoded } = require('express')
+const app=express()
+const port=3000
+const route=require('./routes')
+
+app.use(express.json())
+app.use(urlencoded({extended:true}))
+app.use('/',route)
```

Comment:
ini langsung aja bisa kok 
```suggestion
app.use(route)
```